pgrx/src/fcinfo.rs

```diff
@@ -162,10 +163,12 @@ pub unsafe fn pg_get_nullable_datum(
     fcinfo: pg_sys::FunctionCallInfo,
     num: usize,
 ) -> pg_sys::NullableDatum {
-    let fcinfo = unsafe { fcinfo.as_mut() }.unwrap();
+    let _nullptr_check = ptr::NonNull::new(fcinfo).unwrap();
```

Comment:
Nit: Would this be better as a `.expect("some informative error")` instead of `.unwrap()`?

Reply:
Probably!

Reply:
Thank you! Addressed in 87b253e.